src/components/Sidebar/Participants/ParticipantsList/ParticipantsList.vue

```diff
@@ -23,34 +23,49 @@
 	<div>
 		<ul>
 			<Participant
-				v-for="participant in participantsList"
+				v-for="participant in items"
 				:key="participant.userId"
-				:participant="participant" />
+				:participant="participant"
+				@clickParticipant="handleClickParticipant" />
```

Comment:
click wasn't working?

Reply:
It was working but since I have so many clickable things all over the place I thought it was nice to have more descriptive event names. What do you think?

Reply:
I prefer using the default click as it seems more relevant and shorter, but that's personal preference, so not a big deal :)

Reply:
alright :)